features/websocket.yml

```diff
@@ -1,7 +1,8 @@
-draft_date: 2024-07-11
-name: WebSockets Standard
-description: TODO
+name: WebSockets
```

Comment:
Please keep the filename as `websockets.yml` in line with https://websockets.spec.whatwg.org/ and https://caniuse.com/websockets.

As for the name, the spec is clearly intentional about never saying "web socket" without a space, so I think "WebSockets" as you've use here will work.

Reply:
Renamed.